src/components/Checkbox/Checkbox.module.css

```diff
@@ -2,12 +2,12 @@
 
 .checkbox {
   display: flex;
-  gap: var(--eds-size-1);
+  gap: calc(var(--eds-size-1) / 16 * 1rem);
 }
 
 .checkbox__input {
-  height: 18px;
-  width: 18px;
+  height: 1.125rem;
```

Comment:
Not using a size token here, but making it consistent with the others which use `rem`s for sizing